index.js

```diff
@@ -230,21 +230,6 @@ for (typeName in EXTRA) {
   typeforce[typeName] = EXTRA[typeName]
 }
 
-// async wrapper
-function __async (type, value, strict, callback) {
-  // default to falsy strict if using shorthand overload
-  if (typeof strict === 'function') return __async(type, value, false, strict)
-
-  try {
-    typeforce(type, value, strict)
-  } catch (e) {
-    return callback(e)
-  }
-
-  callback()
-}
-
-typeforce.async = __async
```

Comment:
This was released,  but,  it was never officially mentioned in the README as being a stable part of the API (like it is now).
If anyone complains about the `.async` export missing,  we can re-add it.